templates/pages/links.html

```diff
@@ -8,6 +8,13 @@
 		<p>We gather useful links to tutorials, good practices, etc. </p>
 
 	</div>
-
+<div>
+    <p><a href="http://www.designforfounders.com/web-app-ux/">15 Timeless Rules for Creating Intuitive Web Apps (With Examples)</a></p>
+    <p><a href="goodui.org">Good user interface datastories</a></p>
```

Comment:
Make sure all links have `http://` (or `https://`) in front. If not, the webpage will think this is a relative link and will try to open `http://trackcat.com/goodui.org` (just an example if we actually had a published website).